apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -112,6 +112,11 @@ type OpenTelemetryCollectorSpec struct {
 	// +optional
 	Tolerations []v1.Toleration `json:"tolerations,omitempty"`
 
+	// NodeSelector to schedule OpenTelemetry Collector pods.
+	// This is only relevant to daemonsets, statefulsets and deployments
```

Comment:
Change to: 
> This is only relevant to daemonset, statefulset and deployment mode.

Reply:
That phrase was copied directly from the field above, so I revised both.